### ISSUE TITLE
Why this new logo and colors ? :)

### ISSUE THREAD

**Issue by nvuillam** [oxsecurity/megalinter#1549]
Dear MegaLinter users community,

You probably noticed that the documentation now refers to [**OX Security**](https://www.ox.security/), but... what's OX Security ?
They contacted me some months ago to ask if i'd like MegaLinter to be part of their great initiative.

After some discussions that convinced me about the fact that **we share common values about open-source**, and that **MegaLinter will always remain as free and transparent as it has always been**, I decided that it would be a good way to help my **covid-confinement-hobby project** to **grow** and **be part of something bigger**, **while keeping untouched everything that we love about MegaLinter** :)

A longer and more official announce will be made in the following weeks... including about the **release of v6** that I promise all of you for months now, and that will contain lots of amazing improvements and new features !

**To summarize: Nothing changes except that MegaLinter will grow bigger and better thanks to being part of OX Security family** :)

If you have questions, please feel free to ask them below :)

Nicolas Vuillamy

**Comment by MRDGH2821** [comment]
I can't seem to access OX Secutiry website. It says input password.

I wish to know more about them & other tools they provide/support/mention etc

**Comment by nvuillam** [comment]
@MRDGH2821 the big launch is planned very soon,you'll know everything, and even be able to test it on your public repos if in addition of having nice code with MegaLinter, you also want it (and everything around it) to be secure :)

**Comment by Kurt-von-Laven** [comment]
I think that is a recent development. I don't remember seeing the password prompt when I clicked on the Ox Security link a few days ago.

**Comment by nvuillam** [comment]
> I think that is a recent development. I don't remember seeing the password prompt when I clicked on the Ox Security link a few days ago.

It was a mistake from Dev team, that has been fixed in the following hours :)

**Comment by nvuillam** [comment]
Article explaining everything here :)

https://nicolas.vuillamy.fr/megalinter-sells-his-soul-and-joins-ox-security-2a91a0027628
